airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultCheckConnectionWorker.java

```diff
@@ -84,6 +89,26 @@ public ConnectorJobOutput run(final StandardCheckConnectionInput input, final Pa
           .map(AirbyteMessage::getConnectionStatus)
           .findFirst();
 
+      if (input.getActorId() != null && input.getActorType() != null) {
+        LOGGER.info("has actor and type");
```

Comment:
can this log be removed?

Reply:
Oops! Done in [89622f6](https://github.com/airbytehq/airbyte/pull/20894/commits/89622f620e04088c24e9df7735fc74b2a2ab3442)